content/tbd/STRP.md

```diff
@@ -0,0 +1,38 @@
+---
+created: 2023-12-07
+contributors: Matthew McCarthy
+title: STRP
+url: https://strp.nl/
+locations:
+  - Eindhoven, NL
```

Comment:
format for locations is: city, country in the format like Paris, France or London, UK. I realise this is maybe unclear in documentation guide since the example was UK so I will change it. I think better to spell out country as I think two letter country codes are not always easily recognisable or known without looking it up